src/modules/Popup/Popup.js

```diff
@@ -188,15 +192,25 @@ export default class Popup extends Component {
       }
     }
 
-    if (offset) {
-      if (_.isNumber(style.right)) {
-        style.right -= offset
-      } else {
-        style.left -= offset
-      }
+    return { ...style, ...this.applyOffset(style) }
+  }
+
+  applyOffset = (style) => {
+    const { horizontalOffset, verticalOffset } = this.props
+    return {
+      ...this.applyHorizontalOffset(style, horizontalOffset),
+      ...this.applyVeriticalOffset(style, verticalOffset),
     }
+  }
+
+  applyVeriticalOffset = ({ top, bottom }, offset) => {
+    if (_.isNumber(top)) return { bottom, top: top + offset }
+    return { top, bottom: bottom + offset }
+  }
 
-    return style
+  applyHorizontalOffset = ({ right, left }, offset) => {
+    if (_.isNumber(right)) return { left, right: right + offset }
+    return { right, left: left + offset }
```

Comment:
@bharatzvm, the original code **subtracts** the `horizontalOffset` value from the left/right value, you are adding the offset. Have you made a code change somewhere else that inverts the logic?

Reply:
My Bad, I will update the PR